df.world-data.xml

```diff
@@ -186,8 +186,10 @@
             <int16_t ref-target='plant_raw' />
         </stl-vector>
 
-        <int16_t name="unk_1e4"/>
-        <int16_t name="unk_1e6"/>
+        <int8_t name="unk_1e4" comment="Probably boolean. Probably optionally set only on good and evil regions"/>
```

Comment:
I have a world where `unk_1e4` is set to 0x34

Reply:
That means that the second byte contains a dead_percentage of 52. Note that the change SPLITS unk_1e4 into an 8 bit integer and a boolean (unless you refer to my changed definition).

Reply:
I'm referring to the first byte of these 4 bytes, which is unk_1e4 after your changes. The second byte is either 0x00 or 0x01 - are you sure it's not a boolean?

Reply:
Gah! It means I've screwed up the byte order, so unk_1e4 should be swapped with dead_percentage and unk_1e7 should be swapped with reanimating, i.e.
dead_percentage
unk_1e5
unk_1e6
reanimating